--- config.def.h	2020-10-18 09:26:55.989960672 +0200
+++ config.def.divramod.h	2020-10-18 09:30:29.049957726 +0200
@@ -212,8 +212,11 @@
 	{ ShiftMask,            XK_Insert,      selpaste,       {.i =  0} },
 	{ TERMMOD,              XK_Num_Lock,    numlock,        {.i =  0} },
 	{ XK_ANY_MOD,           XK_F6,          swapcolors,     {.i =  0} },
+  { MODKEY|ShiftMask,     XK_L,           zoom,           {.f = +1} },
+  { MODKEY|ShiftMask,     XK_H,           zoom,           {.f = -1} },
 };
 
+
 /*
  * Special keys (change & recompile st.info accordingly)
  *
